nototools/noto_lint.py

```diff
@@ -988,13 +998,26 @@ def check_cmap_table():
             _check_needed_chars(cmap, tests.get_filter('cmap/script_required'))
 
         if tests.check('cmap/private_use'):
-            privates_in_cmap = {char for char in cmap
-                                if unicode_data.is_private_use(char)}
+            needed_chars = _get_script_required(cmap)
+            pua_filter = tests.get_filter('cmap/private_use')
+            if pua_filter:
+              pua_filter = pua_filter[1].accept
+            def check_pua(char):
```

Comment:
It's hard to understand what check_pua() is supposed to do based on its current name.